website/server/controllers/api-v3/tasks.js

```diff
@@ -634,9 +634,18 @@ api.addTagToTask = {
     if (validationErrors) throw validationErrors;
 
     let taskId = req.params.taskId;
-    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id, { userId: user._id });
+    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id);
+
+    if (!task) {
```

Comment:
Hmm actually... this way the leader will be able to edit tags for all tasks linked to the group, even the ones in the users' accounts. Is it okay?